### PR TITLE
Random fixes

### PR DESCRIPTION
**Pull request by simoheinonen**
Removed unused dependency `justinrainbow/json-schema`, some phpdoc and cs fixes. Doesn't add anything new.

### CODE REVIEW

**Comment by simoheinonen** [comment]
Not sure why the build is failing?

File: ShipwireConnector.php [hunk 1]
```diff
@@ -122,12 +122,17 @@ private function getClient()
 
     /**
      * Send an api request to Shipwire Endpoint
+     *
      * @param string $resource function to be called
-     * @param array $params key value parameters
+     * @param array  $params   key value parameters
```
Comment (sebathi) [inline ShipwireConnector.php]: Please remove these 2 spcaes before and after $params.

**Comment by sebathi** [comment]
If you can remove that 2 spaces i will merge and create a new version :D

**Comment by sebathi** [comment]
Thank you!

**Comment by sebathi** [comment]
Thank you!

**Pull request merged**